packages/picasso/src/Grid/index.ts

```diff
@@ -4,3 +4,4 @@ import { Props } from './Grid'
 
 export { default } from './Grid'
 export type GridProps = OmitInternalProps<Props>
+export { GridSize } from '@material-ui/core/Grid'
```

Comment:
```suggestion
export type { GridSize } from '@material-ui/core/Grid'
```
but probably you need to add this file to `.prettierignore`

Reply:
added :+1: